src/Tec/Plugin.php

```diff
@@ -144,43 +148,32 @@ function option_filter( $fields ) {
 			'3|day'     => esc_html__( '3 days', 'the-events-calendar' ),
 			'1|week'    => esc_html__( '1 week', 'the-events-calendar' ),
 			'2|week'    => esc_html__( '2 weeks', 'the-events-calendar' ),
-			'1|month'   => esc_html__( '1 month', 'the-events-calendar' ),
-			'3|month'   => esc_html__( '3 months', 'the-events-calendar' ),
-			'6|month'   => esc_html__( '6 months', 'the-events-calendar' ),
-			'9|month'   => esc_html__( '9 months', 'the-events-calendar' ),
-			'1|year'    => esc_html__( '1 year', 'the-events-calendar' ),
-			'2|year'    => esc_html__( '2 years', 'the-events-calendar' ),
-			'3|year'    => esc_html__( '3 years', 'the-events-calendar' ),
+			'1'         => esc_html__( '1 month', 'the-events-calendar' ),
+			'3'         => esc_html__( '3 months', 'the-events-calendar' ),
+			'6'         => esc_html__( '6 months', 'the-events-calendar' ),
+			'9'         => esc_html__( '9 months', 'the-events-calendar' ),
+			'12'        => esc_html__( '1 year', 'the-events-calendar' ),
+			'24'        => esc_html__( '2 years', 'the-events-calendar' ),
+			'36'        => esc_html__( '3 years', 'the-events-calendar' ),
 		];
```

Comment:
We have to make sure these are all registered in WP's cron_schedules so they can be identified from a string. e.g.

`tribe-every15mins` is being registered right [here](https://github.com/the-events-calendar/the-events-calendar/blob/feat/slr-support/src/Tribe/Aggregator/Cron.php#L205)